src/components/menu-bottom/MenuBottom.vue

```diff
@@ -1,19 +1,150 @@
+  'data-test': string;
+}
 
 export default Vue.extend({
   name: 'MenuBottom',
+  data: (): {
+    toolDataList: ToolData[];
+    toolSelectedIndex: number;
+    temporaryTool: BaseTool | null;
+  } => ({
+    toolDataList: [
+      {
+        label: 'Pan and Zoom (Hold Ctrl/Meta to turn on)',
+        icon: 'hand-right',
+        tool: ToolPanZoom,
+        'data-test': 'pan-zoom',
+      },
+      {
+        label: 'Add and Remove Single Dot',
+        icon: 'plus-minus',
+        tool: ToolSingleDot,
+        'data-test': 'add-rm',
+      },
+    ],
+    toolSelectedIndex: 0, // Assume that 0 is the pan/zoom tool
+    temporaryTool: null, // Used to hold last tool when ctrl/meta is held
+  }),
+  watch: {
+    toolSelectedIndex(newIndex: number, oldIndex: number) {
+      // Enable or disable pan/zoom depending on tool selected
+      const grapherSvgPanZoom: SvgPanZoom.Instance | undefined
+        = this.$store.state.grapherSvgPanZoom;
+      if (grapherSvgPanZoom === undefined) {
+        return;
+      }
+
+      if (newIndex === 0 && oldIndex !== 0) {
+        grapherSvgPanZoom.enablePan();
+        grapherSvgPanZoom.enableZoom();
+        grapherSvgPanZoom.enableControlIcons();
+      } else if (oldIndex === 0 && newIndex !== 0) {
+        grapherSvgPanZoom.disablePan();
+        grapherSvgPanZoom.disableZoom();
+        grapherSvgPanZoom.disableControlIcons();
+
+        /**
+         * Calculate inverted CTM matrix that is used to convert ClientX/Y to
+         * X/Y of the SVG
+         */
+        const wrapper = document
+          .getElementsByClassName('grapher--wrapper')[0] as SVGGElement;
+        const ctm = wrapper.getScreenCTM();
+        if (!ctm) {
+          throw 'Unable to retrieve wrapper CTM';
+        }
+        const invertedMatrix = ctm.inverse();
+        this.$store.commit('setInvertedCTMMatrix', invertedMatrix);
+      }
+    },
+  },
+  mounted() {
+    this.setTool(this.$data.toolSelectedIndex);
+
+    document.addEventListener('keydown', this.onKeyDown);
+    document.addEventListener('keyup', this.onKeyUp);
+  },
+  beforeDestroy() {
+    document.removeEventListener('keydown', this.onKeyDown);
+    document.removeEventListener('keyup', this.onKeyUp);
+  },
+  methods: {
+    setTool(toolIndex: number): void {
+      this.$data.toolSelectedIndex = toolIndex;
+      const ToolConstructor: ToolConstructor
+        = this.$data.toolDataList[toolIndex].tool;
+      const tool: BaseTool
+        = new ToolConstructor(this.$store);
+      this.$store.commit('setToolSelected', tool);
+    },
+    isCtrl(event: KeyboardEvent): boolean {
+      /* eslint-disable max-len */
+      /**
+       * Capture both Ctrl and Meta key (Cmd on Mac or Windows logo key)
+       * https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values
+       * https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
+       */
+      /* eslint-enable max-len */
+      return event.key === 'Control'
```

Comment:
Since control is going to be used in the pinch zoom gesture in chrome, we should use a different key